class/DatabaseConnection.php

```diff
@@ -0,0 +1,340 @@
+		return $result;
+	}
+
+	/**
+	* To insert data into the database
+	*
+	* @access public
+	* @param  string $table_name, array $data_array
+	* @return object
+	*/
+	public function insert_full($table_name, $data_array)
+	{
+		if($table_name == 'employee')
+		{
+			$q_employee = "INSERT INTO $table_name (first_name, middle_name, last_name, email, 
+				password, prefix, gender, dob, marital_status, employment, employer, photo, 
+				extra_note, comm_id) 
+				VALUES ('{$data_array['first_name']}', '{$data_array['middle_name']}', 
+				'{$data_array['last_name']}', '{$data_array['email']}', 
+				'{$data_array['password']}', '{$data_array['prefix']}', 
+				'{$data_array['gender']}', '{$data_array['dob']}', 
+				'{$data_array['marital_status']}', '{$data_array['employment']}', 
+				'{$data_array['employer']}', '{$data_array['photo']}', 
+				'{$data_array['extra_note']}', '{$data_array['comm_id']}')";
+
+			$result_employee = DatabaseConnection::db_query($q_employee);
+
+			if(!$result_employee)
+			{
+				log_error('Data insertion in employee table');
+				echo "Data insertion into employee table unsuccessful";
+				exit;
+			}
+
+			return self::$conn;
+		}
+		elseif($table_name == 'address')
+		{
+			$q_address = "INSERT INTO $table_name (`emp_id`, `address_type`, `street`, `city`, 
+				`state`, `zip`, `phone`, `fax`) 
+				VALUES
+				({$data_array['emp_id']}, 'residence', '{$data_array['r_street']}', 
+				'{$data_array['r_city']}', '{$data_array['r_state']}', '{$data_array['r_zip']}', 
+				'{$data_array['r_phone']}', '{$data_array['r_fax']}'),
+
+				({$data_array['emp_id']}, 'office', '{$data_array['o_street']}', 
+				'{$data_array['o_city']}', '{$data_array['o_state']}', '{$data_array['o_zip']}', 
+				'{$data_array['o_phone']}', '{$data_array['o_fax']}')";
+
+			$result_address = DatabaseConnection::db_query($q_address);
+
+			if(!$result_address)
+			{
+				log_error('Data insertion in address table');
+				echo "Data insertion into address table unsuccessful";
+				exit;
+			}
+
+			return self::$conn;
+		}		
+	}
+
+	/**
+	* To delete data from the database
+	*
+	* @access public
+	* @param  string $table_name, integer $id
+	* @return void
+	*/
+	public function delete_from_table($table_name, $id)
+	{
+		if($table_name == 'address')
+		{
+			$del_add = "DELETE FROM $table_name WHERE emp_id = " . $id;
+			DatabaseConnection::db_query($del_add);
+		}
+		elseif($table_name == 'employee')
+		{
+			$del_emp = "DELETE FROM $table_name WHERE id = " . $id;
+			DatabaseConnection::db_query($del_emp);
+		}
+	}
+
+	/**
+	* To delete the employee profile picture
+	*
+	* @access public
+	* @param  integer $id
+	* @return void
+	*/
+	public function delete_pic($id)
+	{
+		$sel_pic = "SELECT photo FROM employee WHERE id = " . $id;
+		$pic_object = DatabaseConnection::db_query($sel_pic);
+		$row = DatabaseConnection::db_fetch_array($pic_object);
+		$pic_location = PIC_PATH . $row['photo'];
+		unlink($pic_location);
```

Comment:
unlink generates errors, if file not present. The proper way to do this is check that the file exists before trying to call unlink on it.